msteams-platform/graph-api/rsc/preapproval-instruction-docs.md

```diff
@@ -0,0 +1,238 @@
+
+In this section, you learn to:
+
+* [Create a preapproval without a sensitivity label](#create-the-preapproval-without-a-sensitivity-label) for use with any sensitivity group.
+* [Create a preapproval with a sensitivity label](#create-the-preapproval-with-a-sensitivity-label) for use with specific data.
+
+To create a preapproval, ensure that you have the following information:
+
+1. Teams App ID.
+1. The RSC permissions associated with the app.
+1. Team RSC permission: The sensitivity label ID associated with the sensitivity label you want to link to the preapproval policy.
+1. Global admin privilege in your tenant.
+
+You can use PowerShell cmdlets to create a preapproval. To create a cmdlet, you must gather the information mentioned above with the `New-MgBetaTeamAppPreApproval` command and specify the type of RSC permissions you want to preapprove in the cmdlet.
+
+When you preapprove an app, the following RSC configurations are available:
+
+|Configuration| Description|
+|---|---|
+|`ResourceSpecificApplicationPermissionsAllowedForChats`|When you create a preapproval for RSC permissions for a chat, use this configuration.|
+|`ResourceSpecificApplicationPermissionsAllowedForTeams`|When you create a preapproval for RSC permissions for a team, use this configuration.|
+
+### Create the preapproval without a sensitivity label
+
+When you create your cmdlet to preapprove, you must use the required RSC configuration. If the app you're preapproving has both chat RSC and team RSC permissions, you can include both the configurations.
+
+The following examples show the various ways to build cmdlets for preapproving apps with different RSC permissions, without specifying a sensitivity label:
+
+<br>
+<details>
+
+<summary><b>Preapprove an app with team RSC permissions</b></summary>
+
+```powershell
+New-MgBetaTeamAppPreApproval -TeamsAppId c626ce8b-6d15-4c07-bfb1-a5fd0bc3c20e -ResourceSpecificApplicationPermissionsAllowedForTeams @('OnlineMeeting.ReadBasic.Group')
+```
+
+</details>
+<br>
+<details>
+
+<summary><b>Preapprove an app with chat RSC permissions</b></summary>
+
+```powershell
+New-MgBetaTeamAppPreApproval -TeamsAppId c626ce8b-6d15-4c07-bfb1-a5fd0bc3c20e  -ResourceSpecificApplicationPermissionsAllowedForChats @('OnlineMeeting.ReadBasic.Chat')
+```
+
+</details>
+
+<br>
+<details>
+
+<summary><b>Preapprove an app with both team and chat RSC permissions</b></summary>
+
+```powershell
+New-MgBetaTeamAppPreApproval -TeamsAppId c626ce8b-6d15-4c07-bfb1-a5fd0bc3c20e -ResourceSpecificApplicationPermissionsAllowedForChats @('OnlineMeeting.ReadBasic.Chat') -ResourceSpecificApplicationPermissionsAllowedForTeams @(‘example.permissions.group’)
+```
+
+</details>
+
+> [!NOTE]
+>
+> * If you combine the wrong permissions with the wrong permission type in your cmdlet, the preapproval creation might fail.
+> * The chat RSC permissions must end with `.Chat` and team RSC permissions must end with `.Group`.
```

Comment:
This note is not needed IMO. Customers should only use permissions present in our reference docs.

Reply:
removed in latest commit